draft-ietf-rats-corim.md

```diff
@@ -2417,6 +2496,38 @@ If key verification succeeds for any _key_:
 
 Otherwise, do not add the `addition` ECT to the ACS.
 
+#### Processing Domain Membership {#sec-process-dm}
+
+Domain Membership Triples allow an Endorser (for example, an Integrator) to issue an authoritative statement about the composition of an Attester as a collection of Environments.
+If the Verifier Appraisal policy requires Domain Membership, the Domain Membership Triple is used to match an Attester's reference composition with the actual composition represented by Evidence.
```

Comment:
I am probably missing something, but the policy has “n/a” for members, so I am not sure I understand how policy and DMT are put together.

Reply:
i mean if a Verifier requires DMT, as DMT is an OTI Triple. Perhaps I can qualify this...